### DISCUSSION TITLE
assigning location objects for processes

### DISCUSSION THREAD

**Discussion by bl-young** [NETL-RIC/ElectricityLCI#258] [Q&A]
For future regionalized LCIA, having locations assigned to generation profiles would be nice. Has this been discussed?

Conceivably also the upstream fuel supply chains.

I am working on generating/compiling location objects consistent with existing location objects in openLCA while we also think through LCIA methods.

**Comment by m-jamieson** [comment]
Agreed this is a need, and I've at least thought about it but maybe not discussed it. I think with the regionalization improvements of openLCA since 2.0, the tools are in place to do so. And we have shape files for the production basins for fuel and natural gas, and there are shape files for balancing authority areas. I don't know if it would be better to have relatively large, aggregated areas like that vs a collection of points of power plants within those regions. We don't specify the emissions per power plant in the elci. I guess the question might be which approach might provide more useful/reasonable results?

**Reply by bl-young** [reply]
if i'm understanding correctly, given that all the generation emissions for a region would be assigned to the whole region (e.g. BA), seems better not to worry about specific plants at this point.

There are existing geoJSONs for NERC regions, which I think is the preferred format at this point (e.g. [here](https://geography.ecoinvent.org/#north-american-networks)]

**Reply by bl-young** [reply]
From what i've learned about regionalized LCIA, making sure the location objects align with the LCIA methods is key.

So that means either,
1) the locations should be assigned as states or countries, for which most LCIA methods are likely to have by default
2) some kind of new geojson objects will need to be created for the appropriate regions and incorporated into the LCIA methods

for 1, I wonder if it would be simple enough to include state in the exchange tables for the generation processes, such that each flow would be tied to a specific state (for a balancing authority that crosses multiple states this could increase the exchange table size by 2-3x)
